### PR TITLE
changed the formatting of the cards and the form in the projects page

### PR DESCRIPTION
**Pull request by NathanBlackman**
Changed form and card formatting

### CODE REVIEW

**Comment by TrinityChristiana** [comment]
Noice!

File: scripts/utils/render.js [hunk 1]
```diff
@@ -12,15 +12,17 @@ const projectsCardBuilder = (projectsArr)=> {
...
         </div>
         <ul class="list-group list-group-flush">
-          <li class="list-group-item">${project.name}</li>
           <li class="list-group-item">${project.description}</li>
+          
         </ul>
      </div>`;
   });
 
+  // Projects
```
Comment (austincparker) [inline scripts/utils/render.js]: there's some commented out code here.

Comment (NathanBlackman) [reply scripts/utils/render.js]: Alrighty deleted

**Review by austincparker — APPROVED** [review]
goal!

**Review by pmpurcell — APPROVED** [review]
Looks good to me!

**Pull request merged**